catalog.html

```diff
@@ -0,0 +1,129 @@
+          <label for="natty">MR NATTY</label>
+          <input type="checkbox" name="suavecito" checked>
+          <label for="suavecito">SUAVECITO</label>
+          <input type="checkbox" name="malin" >
+          <label for="malin">MALIN+GOETZ</label>
+          <input type="checkbox" name="murray" >
+          <label for="murray">MURRAY`S</label>
+          <input type="checkbox" name="american" checked>
+          <label for="american">AMERICAN CREW</label>
+        </form>
+        <h2>ГРУППЫ ТОВАРОВ:</h2>
+        <form action="#" class="groups">
+          <input type="radio" name="shave">
+          <label for="shave">БРИТВЕННЫЕ ПРИНАДЛЕЖНОСТИ</label>
+          <input type="radio" name="care" checked>
+          <label for="care">СРЕДСТВА ДЛЯ УХОДА</label>
+          <input type="radio" name="shave">
+          <label for="shave">АКСЕССУАРЫ</label>
+        </form>
+        <form action="#" name="filter-result" class="filter-button">
+          <button name="filter-result">ПОКАЗАТЬ</button>
+        </form>
+      </div>
+      <div class="products">
+        <div class="product-card">
+          <img src="img/01.jpg" width="" height="" alt="">
+          <p class="product-text-card">Набор для путешествий<br>BAXTER OF CALIFORNIA</p>
+          <div class="price">2900 РУБ.</div>
+          <button class="btn" name="buy-btn">КУПИТЬ</button>
+        </div>
+        <div class="product-card">
+          <img src="img/02.jpg" width="" height="" alt="">
+          <p class="product-text-card">Увлажняющий кондиционер<br>BAXTER OF CALIFORNIA</p>
+          <div class="price">750 РУБ.</div>
+          <button class="btn" name="buy-btn">КУПИТЬ</button>
+        </div>
+        <div class="product-card">
+          <img src="img/03.jpg" width="" height="" alt="">
+          <p class="product-text-card">Гель для волос<br>SUAVECITO</p>
+          <div class="price">290 РУБ.</div>
+          <button class="btn" name="buy-btn">КУПИТЬ</button>
+        </div>
+        <div class="product-card">
+          <img src="img/04.jpg" width="" height="" alt="">
+          <p class="product-text-card">Глина для укладки волос<br>AMERICAN CREW</p>
+          <div class="price">500 РУБ.</div>
+          <button class="btn" name="buy-btn">КУПИТЬ</button>
+        </div>
+        <div class="product-card">
+          <img src="img/05.jpg" width="" height="" alt="">
+          <p class="product-text-card">Гель для волос<br>AMERICAN CREW</p>
+          <div class="price">300 РУБ.</div>
+          <button class="btn" name="buy-btn">КУПИТЬ</button>
+        </div>
+        <div class="product-card">
+          <img src="img/06.jpg" width="" height="" alt="">
+          <p class="product-text-card">Набор для бритья<br>BAXTER OF CALIFORNIA</p>
+          <div class="price">3900 РУБ.</div>
+          <button class="btn" name="buy-btn">КУПИТЬ</button>
+        </div>
+        <div class="pages">
+          <button class="page-number" type="button">1</button>
```

Comment:
I'd rather use "a" tag, not "button" here, as we'll go to another page if we click it